khmer/utils.py

```diff
@@ -55,6 +55,10 @@ def check_is_pair(record1, record2):
     elif lhs1 == lhs2 and rhs1.startswith('1:') and rhs2.startswith('2:'):
         return True
 
+    # handle @name seq/1
+    elif lhs1 == lhs2 and rhs1.endswith('/1') and rhs2.endswith('/2'):
```

Comment:
Do we need to check that `rhs1[:-2] == rhs2[:-2]`? I.e., that the read name is equal.

The same check as L47-52 above